demo-app/src/components/aruba/ArubaPin.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, Text, ThemeContext } from 'grommet';
+import { normalizeTheme } from 'grommet-leaflet-core/src/utils';
+import styled from 'styled-components';
+
+const StyledBox = styled(Box)`
```

Comment:
Doesn't have to be solved for here, but might be nice to have an easier way for callers to use this css snippet from grommet-leaflet-core